vars/family_Debian.yml

```diff
@@ -3,8 +3,10 @@ __controlhost_packages_os:
   - iftop
   - htop
   - iotop
-  - python2.7
-  - python-pip
+  - "{{ 'python2.7' if controlhost_python_version == '2' else 'python3.6' }}"
```

Comment:
We should use 3.7 here.

Reply:
Agreed, I think we should still test with Python 3.6 though, since this is the default for CentOS 8.

Reply:
`python3.7` is only available [for Debian Buster](https://packages.debian.org/search?keywords=python3.7). While testing this, I just noticed there isn't even a package `python3.6` for Debian.
So we can either put `python3.5` here ([which is the latest version for stretch](https://packages.debian.org/search?suite=all&section=all&arch=any&searchon=names&keywords=python3.5)), or we go straight with `python3.7` and declare that this role depends on Buster.